test/framework/easyconfig.py

```diff
@@ -3557,6 +3557,55 @@ def test_unexpected_version_keys_caught(self):
 
             self.assertRaises(EasyBuildError, EasyConfig, test_ec)
 
+    def test_resolve_exts_filter_template(self):
```

Comment:
please add docstring:

```python
def test_resolve_exts_filter_template(self):
    """Test for resolve_exts_filter_template function."""

Reply:
I'm against trivial comments, as they don't serve any purpose but making me scroll more... But added anyway.

Reply:
It sort of speaks for itself in this case, true, but it's better to be consistent imho :)